src/app/update-reward/update-reward.component.html

```diff
@@ -0,0 +1,6 @@
+<form class="form-signin" [formGroup]="updateForm" (ngSubmit)="onSubmit()">
+  <h1 class="h3 mb-3 font-weight-normal">to buy</h1>
+  <h1 class="h4 mb-4 font-weight-normal">Are You sure You want to buy this reward?</h1>
```

Comment:
Idea for task: internationalization tool, to not keep hardcoded lang string :)
https://angular.io/guide/i18n